src/style.css

```diff
@@ -0,0 +1,128 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Book.ttf);
+  color: #3d4e61;
+}
+
+.search-image {
+  position: relative;
+}
+
+.search-image::before {
```

Comment:
consider changing the class name to avoid confusion. It is a container, not an image

Reply:
changed name of the class